mapr/README.md

```diff
@@ -16,7 +16,13 @@ The MapR check is included in the [Datadog Agent][2] package but requires additi
 
 - [MapR monitoring][3] is running correctly.
 - You have an available [MapR user][4] (with name, password, UID, and GID) with the 'consume' permission on the `/var/mapr/mapr.monitoring/metricstreams` stream. This may be an already existing user or a newly created user. If you want the user to be called `dd-agent`, create it before installing the Agent.
```

Comment:
```suggestion
- You have an available [MapR user][4] (with name, password, UID, and GID) with the 'consume' permission on the `/var/mapr/mapr.monitoring/metricstreams` stream. This may be an already existing user or a newly created user.
```